packages/permissions/package.json

```diff
@@ -29,10 +31,11 @@
   "scripts": {
     "prepare": "./../../scripts/version.js replace",
     "prebuild": "rimraf dist/**",
-    "build": "yarn build:es && yarn build:cjs",
-    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./src/index.js -o ./dist/permissions.cjs.js",
-    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./src/index.js -o ./dist/permissions.es.js",
-    "build:es:watch": "yarn build:es -w"
+    "build": "yarn build:es && yarn build:cjs && yarn build:typings",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./src/index.ts --dir ./dist --chunkFileNames permissions-[name]-[hash].cjs.js --entryFileNames permissions-[name].cjs.js",
```

Comment:
Why do we need those extra options? Maybe a bad copy/paste? 🤔

Reply:
#746

Reply:
I was confused by that myself and took what the majority had on this PR.